src/barril/units/unit_database.py

```diff
@@ -1396,7 +1387,7 @@ def _DoOperationResultingInNewQuantity(
 # ===================================================================================================
```

Comment:
Remove header (please do a find/replace to remove them all)

Reply:
Done